server/contentCheck/contentCheckRepository.js

```diff
@@ -0,0 +1,178 @@
+const camelize = require('camelize')
+const db = require('../db/db')
+
+const Country = require('../../common/country')
+
+const getExtent = async (countryIso) => {
+  if (!Country.Area.isISOCountry(countryIso)) {
+    return {}
+  }
+
+  const query = `
```

Comment:
why all those temp tables?
couldn't you use the traditional table mappings and create database views?